tests/e2e/default/client-side-transitions/supervisor-user-create.wdio-spec.js

```diff
@@ -0,0 +1,80 @@
+const utils = require('../../../utils');
+const sentinelUtils = require('../../../utils/sentinel');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const userFactory = require('../../../factories/cht/users/users');
+const commonPage = require('../../../page-objects/default/common/common.wdio.page');
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const contactsPage = require('../../../page-objects/default/contacts/contacts.wdio.page');
+const { BASE_URL } = require('../../../constants');
+
+describe('supervisor user create', () => {
```

Comment:
For the naming of the client-side transition files, I [put some suggestions in the PR](https://github.com/medic/cht-core/pull/7988#discussion_r1063634391) and ultimately I think we will want to match that naming in these test files.  (So, something like: `create-user-for-contacts.create-user.wdio-spec.js`.  I know it is long, so if you have any other suggestions let me know!)  

Then, for this describe block it should probably be more generic, like `Create user when adding contact` (since the functionality is technically not limited to supervisors...).

Reply:
Done!